hudi-hadoop-common/src/main/java/org/apache/hudi/common/util/ParquetUtils.java

```diff
@@ -366,6 +382,35 @@ public void writeMetaFile(HoodieStorage storage,
     }
   }
 
+  @Override
+  public byte[] serializeRecordsToLogBlock(StorageConfiguration<?> storageConf,
+                                           List<HoodieRecord> records,
+                                           Schema writerSchema,
+                                           Schema readerSchema,
+                                           String keyFieldName,
+                                           Map<String, String> paramsMap) throws IOException {
+    if (records.size() == 0) {
+      return new byte[0];
+    }
+
+    ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+    HoodieConfig config = new HoodieConfig();
+    paramsMap.entrySet().stream().forEach(entry -> config.setValue(entry.getKey(), entry.getValue()));
+    config.setValue(PARQUET_BLOCK_SIZE.key(), String.valueOf(ParquetWriter.DEFAULT_BLOCK_SIZE));
+    config.setValue(PARQUET_PAGE_SIZE.key(), String.valueOf(ParquetWriter.DEFAULT_PAGE_SIZE));
+    config.setValue(PARQUET_MAX_FILE_SIZE.key(), String.valueOf(1024 * 1024 * 1024));
```

Comment:
Should this value be a static var or something?

Reply:
This PR only moves the code.  I've created a follow-up to revisit these hardcoded config values, HUDI-7755.  My understanding is that for log blocks, current settings are good enough for log scanning.